packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -331,73 +166,33 @@ export class OdspSummaryUploadManager {
                     unreferenced = markUnreferencedNodes ? summaryObject.unreferenced : undefined;
-                    reusedBlobs += result.reusedBlobs;
                     blobs += result.blobs;
                     break;
                 }
                 case api.SummaryType.Blob: {
-                    let hash: string | undefined;
-                    let cachedPath: string | undefined;
-                    // If we are expanding the handle, then the currentPath should exist in the cache as we will get the blob
-                    // hash from the cache.
-                    if (expanded && blobDedupingEnabled) {
-                        hash = this.blobTreeDedupCaches.pathToBlobSha.get(currentPath);
-                        if (hash !== undefined) {
-                            cachedPath = this.blobTreeDedupCaches.blobShaToPath.get(hash);
-                            assert(cachedPath !== undefined, 0x0ab /* "path should be defined as path->sha mapping exists" */);
-                        } else {
-                            // We may not have the blob hash in case its contents were not returned during snapshot fetch.
-                            // In that case just put the current path as cached path as its contents should not have changed
-                            // in expansion flow.
-                            cachedPath = currentPath;
-                        }
+                    if (typeof summaryObject.content === "string") {
+                        value = {
+                            type: "blob",
+                            content: summaryObject.content,
+                            encoding: "utf-8",
+                        };
                     } else {
-                        if (typeof summaryObject.content === "string") {
-                            value = {
-                                type: "blob",
-                                content: summaryObject.content,
-                                encoding: "utf-8",
-                            };
-                        } else {
-                            value = {
-                                type: "blob",
-                                content: Uint8ArrayToString(summaryObject.content, "base64"),
-                                encoding: "base64",
-                            };
-                        }
-                        if (blobDedupingEnabled) {
-                            hash = await hashFile(IsoBuffer.from(value.content, value.encoding));
-                            cachedPath = this.blobTreeDedupCaches.blobShaToPath.get(hash);
-                        }
+                        value = {
+                            type: "blob",
+                            content: Uint8ArrayToString(summaryObject.content, "base64"),
+                            encoding: "base64",
+                        };
                     }
+
                     (summaryObject as any).content = undefined;
```

Comment:
Do we still need this? I think it was done because we were caching tree and did not want to cache binary payloads.
Can you please take a look?

Reply:
No, removed it. Thanks for catching but it was harmless.